fleet_adapter_ecobot/EcobotClientAPI.py

```diff
@@ -55,7 +58,7 @@ def localize(self, init_point:str, map_name:str, rotate=False):
         if rotate: # The robot will rotate to improve localization
             url = self.prefix + f"/gs-robot/cmd/initialize?map_name={map_name}&init_point_name={init_point}"
         else: # The specified init point must be accurate
-            url = self.prefix + f"/gs-robot/cmd/initialize_directly?map_name={map_name}&init_point_name=?{init_point}"
```

Comment:
Did the endpoint in the API change? Or what the original incorrect?

Reply:
The original was incorrect, tested with this and it works.